ucdavis/sitefarm-distro-template
# ucdavis/sitefarm-seed#145 Added docksal config.

To test this:

1. Turn off MAMP (or whatever local development environment you're using)

2. Install Docksal:
http://docksal.readthedocs.io/en/master/getting-started/env-setup/

3. Make a fresh git clone of branch ```issue-145``` of ucdavis/sitefarm-distro-template

4. Run: ```$ fin init``` in the project root

5. Go to: 
http://sitefarm-distro.local

You should see SiteFarm.
